Source/WebCore/html/track/VTTCue.h

```diff
@@ -112,7 +113,12 @@ class VTTCueBox : public TextTrackCueBox {
 
 // ----------------------------
 
-class VTTCue : public TextTrackCue {
+class VTTCue
+    : public TextTrackCue
+#if !RELEASE_LOG_DISABLED
+    , private LoggerHelper
+#endif
```

Comment:
Technically, you don't need to inherit from `LoggerHelper`; You just need to implement the methods LoggerHelper needs.